Source/public/css/style.css

```diff
@@ -77,3 +81,33 @@ body {
 	margin: 0.5%;
 	float: right;
 }
+
+.first-col {
+  margin-bottom: 20px;
+}
+
+.col-xl-6 {
+  margin-top: 20px;
+}
+
+#tempChart, #lightChart, #gassChart {
+  margin-top: 5px;
+}
+
+.heading-margin {
+  margin-top: 15px;
+}
+
+.logo {
+  margin: 0 50px;
+}
+
+.flag {
```

Comment:
Vlaggen missen alleen nog in de navbar

Reply:
Hoe kan het dat die ineens verwijderd zijn dan? Ik fix t vanaaf

Reply:
Het is jouw ook niet opgevallen dat ze weg waren, anders had je ze er wil weer ingezet (neem ik aan). Op diezelfde manier is het dus verwijderd en goedgekeurd door de reviewer.